### ISSUE TITLE
sdk: ko/wizard

### ISSUE THREAD

**Issue by cgchoffman** [Komodo/KomodoEdit#2317]
This ask must have missed some migration step when the wizard sdk got moved/changed.  It doesn't work and doesn't seem to follow the general UI SDK.

**Comment by Naatan** [comment]
What do you mean "doesnt work"? Doesn't our first start wizard use this?

**Comment by cgchoffman** [comment]
No, the first start wizard bypasses the sdk by using ko/dom.  `wizard.addPage` doesn't work.  `wizard.Open` should still work.  It uses a `type` field which is what I think I originally used to identify what type of element we were making but that has since been changed to `name`.  That's all i've found so far.

When `addPage` does work, it should support adding a `ko/ui/wizardpage`.

**Comment by Naatan** [comment]
So ko/wizard isnt used by anything?

**Comment by cgchoffman** [comment]
I haven't looked. I doubt it though.  It doesn't work.

**Comment by Naatan** [comment]
Doesn't look like it (searched for it in our codebase). Let's remove this from Komodo until we have a use for it. I don't see any point in spending dev cycles on developing/fixing an SDK we don't use.

**Comment by cgchoffman** [comment]
OK but we should add it back at some point.  We could build a helpful API around the MDN wizard implementation.

**Comment by Naatan** [comment]
The MDN version is too limiting imo. I agree a proper wizard SDK would be a useful tool to have, but it's not worth implementing until we have a good use case for one.

**Comment by cgchoffman** [comment]
> The MDN version is too limiting imo.

Yes, that is why we would wrap it, like we did with all the other UI SDKs.
